### ISSUE TITLE
Mention the use of dependabot

### ISSUE THREAD

**Issue by Stephen-Gates** [theodi/toolbox#22]
[dependabot](https://dependabot.com/) is used across the toolbox to keep dependencies up to date. Perhaps it's worth a mention in the readme / wiki?

**Comment by Floppy** [comment]
Good idea. I only added it recently, but I'd say it's passed its trial, it's being really useful :)
